packages/client/lib/commands/HTTL.ts

```diff
@@ -0,0 +1,11 @@
+import { RedisArgument, ArrayReply, NumberReply, Command } from '../RESP/types';
+import { RedisVariadicArgument, pushVariadicArgument } from './generic-transformers';
+
+export default {
+  FIRST_KEY_INDEX: 1,
+  IS_READ_ONLY: true,
+  transformArguments(key: RedisArgument, fields: RedisVariadicArgument) {
+    return pushVariadicArgument(['HTTL', key], fields);
+  },
+  transformReply: undefined as unknown as () => ArrayReply<NumberReply>
```

Comment:
the reply is array of

> Integer reply: -2 if no such field exists in the provided hash key, or the provided key does not exist.
Integer reply: -1 if the field exists but has no associated expiration set.
Integer reply: the TTL in seconds.

should we create an "enum" (not the TS one) for the -2 and -1 values? IDK if I like having them anymore..

Reply:
an enum can have value to refrer to negative values by name.  but isn't that what I did before ala

```javascript
export const HASH_FIELD_EXPIRETIME = {
  /** The field does not exist /
  FIELD_NOT_EXISTS: -2,
  /* The field has no expiration time */
  NO_EXPIRATION_TIME: -1,
} as const;
```